util/src/main/java/org/datacommons/util/TmcfCsvParser.java

```diff
@@ -191,24 +235,58 @@ public void process(CSVRecord dataRow) {
           instanceValues.addTypedValues(newTypedValue.build());
         } else if (typedValue.getType() == Mcf.ValueType.TABLE_COLUMN) {
           // Replace column-name with cell-value
-          int columnIndex = toColumnIndex(typedValue.getValue());
-          assert columnIndex < dataRow.size() : "Unexpected inconsistent row " + dataRow.toString();
+          McfParser.SchemaTerm term = McfParser.parseSchemaTerm(typedValue.getValue(), errCb);
+          if (term == null) {
+            continue;
+          }
+          if (term.type != McfParser.SchemaTerm.Type.COLUMN) {
+            addLog(
+                Debug.Log.Level.LEVEL_ERROR,
+                "CSV_UnexpectedNonColumn",
+                "Unexpected non-column "
```

Comment:
i'm not sure if this message is completely clear (what does non-column mean?). we might want to err on being more verbose with our errors.

Reply:
Tried to reword.

Did not really focus on the error messages in this PR, just trying to plumb the logCtx everywhere... but yeah, we need to do a pass on every call that adds logEntry and ensure it is verbose enough...

Reply:
One more idea is, since we have a distinct counter name for these errors, we can put a lot more verbose details in a central doc page, like:

https://docs.datacommons.org/errors#<counter_name>  and link to it.

Reply:
oh yes, that would be cool. but i like the idea of revisiting all the error messages prior to launch

Reply:
Agreed. In fact, might be good for another pair of eyes to do a pass on the strings passed into all addLog() calls....